go.mod

```diff
@@ -154,6 +155,7 @@ require (
 )
 
 replace (
+	github.com/ChainSafe/go-schnorrkel v1.0.0 => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
```

Comment:
Why?

Reply:
because the mev package uses the `v0.0.0-20200405005733-88cbf1b4c40d` version and refuses to compile with the `v1.0.0` one